trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilitySnapshot.js

```diff
@@ -14,63 +14,65 @@ const mapUkefExposureValue = require('./mapUkefExposureValue');
 const mapFacilityValueExportCurrency = require('./mapFacilityValueExportCurrency');
 
 /**
- * Maps existing facility to the facility used in TFM API.
- * Note: This implimentation is called where it modifies the facility snapshot
- * to have values not consistent with the facility snapshot in the database.
+ * Maps the existing facility snapshot in the database to the facility snapshot used in TFM API.
+ * Note: This implementation modifies the facility snapshot to have values not consistent with the facility snapshot in the database.
+ * In particular, this is a live object that updates e.g. when amendments are added in TFM.
  */
-const mapFacility = (f, facilityTfm, dealDetails, facilityFull) => {
+const mapFacilitySnapshot = (facility, dealDetails) => {
```

Comment:
No functionality of this file has changed, just how we are naming things. I have renamed:

`f` -> `facilitySnapshot`
`facilityFull` -> `facility`
`facility` (this previously was the deep cloned facility snapshot) -> `f`